src/test/java/com/salesforce/einsteinbot/sdk/client/ClientApiWireMockTest.java

```diff
@@ -309,4 +335,15 @@ private void stubStatusResponse(String responseBodyFile) {
     );
   }
 
+  private void stubVersionsResponse(String responseBodyFile) {
```

Comment:
Can we add tests for the case when Versions endpoint returns ErrorResponse?

Reply:
Added this